File: xiuxian-v2/_tmp/gen.py
import os
B="D:/xiuxian/xiuxian/xiuxian-v2"
def w(p,c):
  fp=os.path.join(B,*p.split("/"))
  os.makedirs(os.path.dirname(fp),exist_ok=True)
  open(fp,"w",encoding="utf-8").write(c)
  print("wrote "+p)

File: xiuxian-v2/_tmp/gen.js
const fs=require("fs");
const path=require("path");
const B="D:/xiuxian/xiuxian/xiuxian-v2";
function w(f,c){const fp=path.join(B,...f.split("/"));path.mkdirSync(path.dirname(fp),{recursive:true});fs.writeFileSync(fp,c,"utf8");console.log("wrote "+f);}
